scripts/parts.py

```diff
@@ -50,10 +51,11 @@
     args.revision = args.revision.upper()
     if args.revision == 'ALL':
         args.active = False
-    if args.levels_testing.lower() == 'none' or args.levels_testing.lower() == 'false':
-        args.levels_testing = False
-    elif args.levels_testing == 'levels.tst':
-        args.levels_testing = os.path.join(mc.test_data_path, 'levels.tst')
+    if type(args.levels_testing) == str:
```

Comment:
Why is this `type()` test needed? Is `args.levels_testing` equal to `None` if it's not specified? If so the test should be more specific: `if args.levels_testing is not None:`